app/util/network.py

```diff
@@ -100,7 +101,17 @@ def encode_body(self, body_decoded):
         """
         return json.dumps(body_decoded)
 
-    def _request(self, method, url, data=None, should_encode_body=True, error_on_failure=False, *args, **kwargs):
```

Comment:
@wjdhollow Ah yeah, I decided not to do that here because of the union types (e.g., `dict|str|bytes|None`) that would force me to mix between docstring and annotations. I want to add mypy/typing to this repo at some point so that we can do more complex type annotations.